### PR TITLE
Hotfix 1174.1

### PR DESCRIPTION
**Pull request by AlexNi245**

### CODE REVIEW

File: packages/messenger-widget/src/hooks/messages/useMessage.test.tsx [hunk 1]
```diff
@@ -155,133 +155,6 @@ describe('useMessage hook test cases', () => {
...
-            const addMessageResult = await waitFor(() =>
-                result.current.addMessage('max.eth', message),
-            );
-
-            expect(addMessageResult).toEqual({
-                isSuccess: false,
-                error: 'Message is empty',
-            });
-        });
-        it('should add message', async () => {
```
Comment (Bhupesh-mfsi) [inline packages/messenger-widget/src/hooks/messages/useMessage.test.tsx:221]: Wrong test has been removed. Need to remove this test **should not add empty message**

File: packages/messenger-widget/src/hooks/messages/useMessage.tsx [hunk 2]
```diff
@@ -278,10 +278,6 @@ export const useMessage = () => {
         const contact = normalizeEnsName(_contactName);
         //If a message is empty it should not be added
```
Comment (Bhupesh-mfsi) [inline packages/messenger-widget/src/hooks/messages/useMessage.tsx:279]: This comment is also not needed now.

**Review by Bhupesh-mfsi — CHANGES_REQUESTED** [review]
Please check comments

**Review by Bhupesh-mfsi — APPROVED** [review]
